promgen/templates/promgen/vue/silence_table_modal.html

```diff
@@ -0,0 +1,61 @@
+{% load i18n %}
+{% load promgen %}
+<div id="silenceTableModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="silenceModalLabel">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" @click="state.show = false" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="silenceModalLabel">Silence</h4>
+      </div>
+      <div class="modal-body" style="padding:10px 30px;">
+        
+        <div class="row mb-4">
+          <div class="col-md-12">
+            <div style="display: flex;align-items: center;">
+              <input type="text" placeholder="inpout filter matcher label" v-model="form.label" class="form-control">
+              <input type="text" placeholder="=~" style="max-width: 50px; text-align: center;" class="form-control ml-2 mr-2"  disabled>
+              <input type="text" placeholder="input filter matcher value" v-model="form.value" class="form-control">
+              <button type="button" class="btn btn-primary ml-2" @click="addFilterLabel">&plus;</button>
+            </div>
+          </div>
+        </div>
+
+
+        <div class="row mb-4">
+            <div class="col-md-12">
+              <div class="labels">
+                <ul class="list-inline promgen-labels-list">
+                  <li v-for="item in state.labels" :key="`${item.label}-${item.value}`">
+                    <div class="promgen-label-target">
+                      <span>[[item.label]]=~[[item.value]]</span>
+                      <span @click="removeFilterLabel(item.label, item.value)" aria-hidden="true" class="promgen-label-close">&times;</span>
+                    </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        
+        <table class="table table-bordered table-condensed table-responsive">
+          <thead>
+            <tr>
+                <th>Starts</th>
+                <th>Ends</th>
+                <th>Matchers</th>
+                <th>Comment</th>
+                <th>Created by</th>
+                <th>Action</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <template v-for="silence in filteredSilences" :key="silence.id">
+              <silence-row :silence="silence" label-color="warning" @matcher-click="addFilterLabel" />
```

Comment:
I will do an in-depth review later since I'm currently busy, but this is something I spotted while having a quick look.

The `warning` color is something we normally use when a button **creates** a silence. In this case, we are **filtering** silences, so please use a different color like `info`, for instance.

```suggestion
              <silence-row :silence="silence" label-color="info" @matcher-click="addFilterLabel" />
```